scripts/criu-ns

```diff
@@ -70,7 +70,7 @@ def _wait_for_process_status(criu_pid):
         try:
             (pid, status) = os.wait()
             if pid == criu_pid:
-                return os.waitstatus_to_exitcode(status)
+                return os.WEXITSTATUS(status)
```

Comment:
This hunk reverts the commit https://github.com/checkpoint-restore/criu/commit/7177938e60

First thing, you should not revert something without a good explanation why. And it seems you revert is wrong as it reintroduces the original error we tried to fix in the above commit.

Second, I see that you do this revert due to test error https://github.com/checkpoint-restore/criu/pull/2114#issuecomment-1476129552 that os.waitstatus_to_exitcode is not available.

To fix it you have two options:

a) You can opencode os.waitstatus_to_exitcode in criu-ns to fix this error e.g. like here:
https://github.com/python/cpython/issues/84275#issuecomment-1093867008
```
if os.WIFSIGNALED(status):
        self.returncode = -os.WTERMSIG(status)
    elif os.WIFEXITED(status):
        self.returncode = os.WEXITSTATUS(status)
    elif os.WIFSTOPPED(status):
        self.returncode = -os.WSTOPSIG(status)
    else:
        raise Exception("... put your favorite error message here ...")
```
b) check python version before testing criu-ns test and skip it if python does not have os.waitstatus_to_exitcode.

The latter seems better for me.

Reply:
Apologies, I was unaware of the changes made in `7177938e60`. I just corrected it with the 'a' approach you mentioned. Thanks!